shared/store/wallet.ts

```diff
@@ -98,16 +95,6 @@ export const setTokenSymbol = (t: AddressToken): WalletToken => {
   }
 }
 
-const associateTokens = (tokens: TokenData[]): AssociatedToken => {
-  const result: AssociatedToken = {}
-  tokens.forEach(token => {
-    if (token.isDAT) {
```

Comment:
We are filtering out `isDAT` because the API returns multiple displaySymbol and some of it are not the DAT

Here's my 2 cents:
 - If this will be removed, then we need to update `tokenSelectorByDisplaySymbol` to only return `isDAT` or else it would return incorrect ids

- Also if this filtering will be removed on this level, we always have to be mindful when using `allTokens` to always use `isDAT` unless oceanAPI will update it on their end to always filter it out



<img width="745" alt="image" src="https://user-images.githubusercontent.com/13292662/165695160-ab09a261-b97a-44dc-9187-90663789e29c.png">

Reply:
thanks for the context

Reply:
updated